main.go

```diff
@@ -5,13 +5,13 @@ import (
 	"flag"
 	"fmt"
 	"errors"
-	"gopkg.in/yaml.v2"
 
 	"b01901143.git/sample-go-app/config"
 )
 
 type options struct {
 	configPath    string
+	dynamic    	  bool
```

Comment:
It looks like this code hasn't been formatted with `gofmt`.  This would fail tests in a repo that ensures gofmt is used

You can do this manually by running `gofmt -w .` which will format the code in place

Alternatively, the easier way to do this is use an IDE or text editor that will automatically run gofmt on save

Reply:
Done.
Also set up gofmt package for my sublime editor.